src/plugins/subscription-manager.py

```diff
@@ -159,6 +159,23 @@ def warnOrGiveUsageMessage(conduit):
             conduit.info(2, msg)
 
 
+def init_hook(conduit):
+    """ Hook for disabling system repositories (= repositories which are
+        not mangaged by subscription-manager will NOT be used) """
+
+    disable_system_repos = conduit.confBool('main', 'disable_system_repos', default=False)
+
+    if disable_system_repos:
+        disable_count = 0
+        repo_storage = conduit.getRepos()
+        for repo in repo_storage.repos.values():
+            if os.path.basename(repo.repofile) != "redhat.repo" and repo.enabled == True:
```

Comment:
The condition should be: `repo.enabled is True`

Reply:
Thank you!